CONTRIBUTING.md

```diff
@@ -170,11 +206,11 @@ To see what resources will be deployed for a live service, check the
 `test-resources.json` ARM template files in the service you wish to deploy for
 testing, for example `sdk\keyvault\test-resources.json`.
 
-To deploy live resources for testing use the steps documented in [`Example 1 of New-TestResources.ps1`](eng/common/TestResources/New-TestResources.ps1.md#example-1)
-to set up a service principal and deploy live testing resources.
+To deploy live resources for testing use the steps documented in [`Example 1 of New-TestResources.ps1`](eng/common/TestResources/New-TestResources.ps1.md#example-1) to set up a service principal and deploy live testing resources.
+
+The script will provide instructions for setting environment variables before running live tests.
```

Comment:
How does the script provide instructions for setting env variables? Does it print it out to the console?

Reply:
This line is a mistake. I've removed it.